feature/office/src/main/java/com/iteneum/office/presentation/viewmodel/OfficeViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.iteneum.office.presentation.viewmodel
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import javax.inject.Inject
+
+/**
+ * This class is the viewModel corresponding to the Office screen
+ * @author Yaritza Moreno
+ */
+@HiltViewModel
+class OfficeViewModel @Inject constructor(): ViewModel() {
+
+    private fun onClick(){
```

Comment:
This function shouldn't be private.
Please be more descriptive on which on click is executed